app/scripts/modules/kubernetes/src/v2/manifest/showManifestDetails.component.ts

```diff
@@ -1,25 +1,39 @@
-import { module, IComponentOptions, IController } from 'angular';
+import { IComponentOptions, IController, IScope, module } from 'angular';
 import { trim } from 'lodash';
-import { ReactInjector } from '@spinnaker/core';
+
+import { IManifest, ReactInjector } from '@spinnaker/core';
+import { KubernetesManifestService } from './manifest.service';
 
 const supportedKinds = ['deployment', 'replicaset'];
 
 class KubernetesShowManifestDetails implements IController {
-  public manifest: any;
+  public manifestContents: any;
+  public manifest: IManifest;
   public accountId: string;
 
+  public constructor(kubernetesManifestService: KubernetesManifestService,
+                     private $scope: IScope) {
+    // not good
+    const app = $scope.$parent.$parent.$parent.$parent.application;
```

Comment:
Sorry @danielpeach  :sweat:

Reply:
Oh, you just need to pass `application` into this component from the execution details controller. `application` should live on that controller's scope.